homeassistant/components/sensor/mopar.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, session):
         self.vehicles = []
         self.vhrs = {}
         self.tow_guides = {}
+        self.update()
```

Comment:
By passing `True` as second argument to `add_devices`, Home Assistant will call `update()` before adding it to Home Assistant. So I'm not 100% sure how this can fix it 🤔

Reply:
(I do believe you if you say that it fixes it, I just don't understand how)